FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/PlottingMethod.java

```diff
@@ -0,0 +1,14 @@
+package hec.ensemble.stats;
+
+public class PlottingMethod {
+    float[] method(float[] values) {
+        return values;
+    }
+    public PlottingMethod getMethod(PlottingType type) {
+        PlottingMethod plotting = new PlottingMethod();
+        if (type.getName().equalsIgnoreCase("Weibull")) {
```

Comment:
median is not implemented right? if the type is median you should do something  unless the median is the equivalent of new PlottingMethod(), and i am just missing that math somewhere..

consider a switch case with each element in the enum specified and the default case handled.

Reply:
added Median case and class and switched to switch